mon-pix/app/styles/app.scss

```diff
@@ -108,3 +108,14 @@ input[type='radio'] {
 input:invalid {
   box-shadow: none;
 }
+
+/* Remove the blue background on chrome autofill */
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus textarea:-webkit-autofill,
+textarea:-webkit-autofill:hover textarea:-webkit-autofill:focus,
+select:-webkit-autofill,
+select:-webkit-autofill:hover,
+select:-webkit-autofill:focus {
+  -webkit-box-shadow: 0 0 0px 1000px #ffffff inset !important;
+}
```

Comment:
quel rapport avec le titre du commit "Remove obsolete fonts, use Roboto as default font" ?

Reply:
plusieurs lignes modifiées sont dans ce même cas. Pourquoi ne pas faire un commit séparé pour chaque chose distincte faite ? Exemple : `Update margins on several pages`, `Update color use`, `Remove the blue background on chrome autofill` et `Remove useless css classes`